src/control/server/config.go

```diff
@@ -259,18 +256,22 @@ func (c *configuration) cmdlineOverride(opts *cliOptions) {
 		if opts.Cores > 0 {
 			c.Servers[i].Cpus, _ = setNumCores(int(opts.Cores))
 		}
+		// Targets should override Cores if specified in cmdline or
+		// config file.
+		if opts.Targets > 0 {
+			c.Servers[i].Cpus, _ = setNumCores(opts.Targets)
+		} else if c.Targets > 0 {
+			c.Servers[i].Cpus, _ = setNumCores(c.Targets)
```

Comment:
Thanks.
then when user either give "-c" or "-t", it will be transferred to "-t" to daos_io_server.
a minor point is probably can just give a ""-c" option is deprecated, please use "-t" instead" hints when "-c" is specified (but still can pass -t to daos_io_server), so we can remove -c some time later after user all know that.
And seems "-t" can be same as "-c" as a per-server parameter.

Reply:
thanks for clarification, will add deprecated message and replace per-server "cores" with "targets" in config and remove global "targets" parameter